README.md

```diff
@@ -4,221 +4,501 @@
 [![GoDoc](https://godoc.org/github.com/v3io/frames?status.svg)](https://godoc.org/github.com/v3io/frames)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-V3IO Frames is a high-speed server and client library for accessing time-series (TSDB), NoSQL, and streaming data in the [Iguazio Data Science Platform](https://www.iguazio.com).
+V3IO Frames (**"Frames"**) is a multi-model open-source data-access library, developed by Iguazio, which provides a unified high-performance DataFrame API for working with data in the data store of the [Iguazio Data Science Platform](https://www.iguazio.com) (**"the platform"**).
 
-## Documentation
+#### In This Document
 
-Frames currently supports 3 backends and basic CRUD functionality for each.
+- [Client Python API Reference](#client-python-api-reference)
+- [Contributing](#contributing)
+- [LICENSE](#license)
 
-Supported Backends:
-1. TSDB
-2. KV
-3. Stream
-4. CSV - for testing purposes
+<a id="client-python-api-reference"></a>
+## Client Python API Reference
 
+- [Overview](#overview)
+- [User Authentication](#user-authentication)
+- [Client Constructor](#client-constructor)
+- [Common Client Method Parameters](#client-common-method-params)
+- [The create Method](#method-create)
+- [The write Method](#method-write)
+- [The read Method](#method-read)
+- [The delete Method](#method-delete)
+- [The execute Method](#method-execute)
+
+<a id="api-reference-overview"></a>
+### Overview
+
+To use Frames, you first need to import the **v3io_frames** Python library.
+For example:
+```python
+import v3io_frames as v3f
+```
+Then, you need to create and initialize an instance of the `Client` class; see [Client Constructor](#client-constructor).
+You can then use the client methods to perform different data operations on the supported backend types:
+
+<a id="client-methods"></a>
+#### Client Methods
+
+The `Client` class features the following methods for supporting basic data operations:
+
+- [`create`](#method-create) &mdash; create a new NoSQL or TSDB table or a stream ("the backend").
```

Comment:
Create for NoSQL is not supported

Reply:
@talIguaz Good catch. I forgot to update the method descriptions in the overview. => I'll prepare an update to the descriptions of all the methods in the overview.